com.ibm.streamsx.topology/opt/python/packages/streamsx/topology/topology.py

```diff
@@ -83,6 +84,51 @@ def isolate(self):
         op.addInputPort(outputPort=self.oport)
         oport = op.addOutputPort()
         return Stream(self.topology, oport)
+    
+    def parallel(self, width):
+        """
+        Marks operator as parallel output with width
+        :param: width
+        :returns: Stream
+        """
+        iop = self.isolate()
+               
+        op2 = self.topology.graph.addOperator("$Parallel$")
+        op2.addInputPort(outputPort=iop.getOport())
+        oport = op2.addOutputPort(width)
+        return Stream(self.topology, oport)
+
+    def endParallel(self):
+        """
+        Marks end of operators as parallel output with width
+        :returns: Stream
+        """
+        op = self.topology.graph.addOperator("$EndParallel$")
+        op.addInputPort(outputPort=self.oport)
+        oport = op.addOutputPort()
+        endP = Stream(self.topology, oport)
+        return endP.isolate()
+
+    def union(self, streamSet):
```

Comment:
Looks good, but I'm reticent to merge until the fix for the `removeOperator` bug has been delivered. Maybe this should be included in a separate pull request?